examples/gallery/embellishments/ternary.py

```diff
@@ -0,0 +1,35 @@
+"""
+Ternary diagram
+---------------------------------------
+
+To plot circles (diameter = 0.1 cm) on a 10-centimeter-wide ternary diagram at the
+positions listed in the file ternary.txt, with default annotations and
+gridline spacings, using the specified labeling.
```

Comment:
```suggestion
To plot circles (diameter = 0.1 cm) on a 10-centimeter-wide ternary diagram at the
positions listed in the sample dataset `rock_compositions`, with default annotations and
gridline spacings, using the specified labeling.
```
Shouldn't this reference the sample dataset, not the original filename?

Reply:
Do you plan to add here more explanation? Currently, this is not a complete sentence.

Reply:
Yes will update this the next few days.